src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -154,7 +154,8 @@ describe('Visual refresh toolbar only', () => {
       })
     );
 
-    test('first opened drawer should be closed when active drawers can not be shrunk to accommodate it (1400px)', () => {
+    test(
```

Comment:
I just realized that this test suite didn't run earlier due to an extra wrap with an arrow function

Reply:
Thanks